utils/v10.ts

```diff
@@ -114,6 +115,20 @@ export function isInteractionButton(component: APIButtonComponent): component is
 	return ![ButtonStyle.Link, ButtonStyle.Premium].includes(component.style);
 }
 
+// Modal
+
+/**
+ * A type-guard check for modals submit interactions
```

Comment:
"type guard" is not hyphenated.

```suggestion
 * A type guard check for modals submit interactions
```

Looks like it is not correct elsewhere too... that is unfortunate.